Common/Tests/MockVsTests/MockVs.cs

```diff
@@ -482,9 +489,10 @@ out _monSelCookie
 
         private CompositionContainer CreateCompositionContainer() {
             var container = new CompositionContainer(CachedInfo.Catalog);
-            container.ComposeExportedValue<MockVs>(this);
-            var batch = new CompositionBatch();
 
+            var batch = new CompositionBatch();
+            batch.AddExportedValue(this);
+            batch.AddExportedValue(typeof(SComponentModel).FullName, this);
             container.Compose(batch);
```

Comment:
This (and related SComponentModel changes) is totally the wrong fix - it should only be a VS service.